storage/innobase/include/os0enc.h

```diff
@@ -83,6 +83,14 @@ class Encryption {
   information version. */
   static constexpr char KEY_MAGIC_V3[] = "lCC";
 
+#ifdef XTRABACKUP
+  static constexpr char KEY_MAGIC_PS_V3[] = "PSC";
+  /* To only  process crypt redo log generated by PS with encryption='n'*/
+  static constexpr uint KEYRING_INFO_MAX_SIZE = 102;
+  /*to know if encryption is kerying or 'n' */
+  static constexpr uint CRYPT_SCHEME_UNENCRYPTED = 0;
```

Comment:
const is fine

Reply:
Can you please point me to where this is defined on PS?

Reply:
https://github.com/percona/percona-server/blob/afc5f453aff448993fd5780fb7e9fb281b441978/storage/innobase/include/fil0rkinfo.h#L4